docs/README.md

```diff
@@ -1,5 +1,7 @@
 # Gitea: Docs
 
+edit!
```

Comment:
What's this?

Reply:
Just a test to confirm that labeling works.

Reply:
Reverted.